ipfs-api-examples/examples/check_file_local.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 rust-ipfs-api Developers
+//
+// Licensed under the Apache License, Version 2.0, <LICENSE-APACHE or
+// http://apache.org/licenses/LICENSE-2.0> or the MIT license <LICENSE-MIT or
+// http://opensource.org/licenses/MIT>, at your option. This file may not be
+// copied, modified, or distributed except according to those terms.
+//
+
+use futures::StreamExt;
+use ipfs_api_examples::ipfs_api::{Error as IpfsError, GlobalOptions, IpfsApi, IpfsClient};
+use ipfs_api_prelude::{request::Ls, response::LsResponse, ApiError, BackendWithGlobalOptions};
```

Comment:
Realizing now that `ApiError` is not exposed, but can we try to import everything from `ipfs_api_examples::ipfs_api`?